CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html
 
 ### Changed
 - Grouped semgrep CLI options and added constraints when useful (e.g. cannot use `--vim` and `--emacs` at the same time)
+- CFGs now track their entry point, and dataflow uses this to build a workset by DFS
```

Comment:
In the changelog I would personally only describe what effect this has from the user perspective, which is that unreachable statements are no longer visited by the Dataflow engine.